src/components/mainMenu/MainMenu.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import {
+  Link,
+} from 'react-router-dom';
+
+const MainMenu = () => (
+  <div className="arcade-cabinet">
+    <div className="score" />
+    <div className="message-box" />
```

Comment:
Are the above classes doing anything or did you work these into your angerNoodle game? They might be redundant now